spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'stations' do
+  include_context 'API client'
+
+  it 'registers a station', vcr: { cassette_name: 'stations/register_success' } do
+    client.endpoint = 'https://api.openweathermap.org/data/3.0'
```

Comment:
Let's extend the context to take options like so:

```ruby
include_context 'API client', endpoint: 'https://api.openweathermap.org/data/3.0'
```

Reply:
Does this work out-of-the-box? It seems like we can only pass a block https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-context#declare-a-shared-context,-include-it-with-%60include-context%60-and-extend-it-with-an-additional-block